### ISSUE TITLE
Observe other games

### ISSUE THREAD

**Issue by Zomis** [Cardshifter/Cardshifter#110]
When two players are playing, and you're not one of them, it would be nice to have an observe feature so that you can watch while they are playing

**Comment by Phrancis** [comment]
@Zomis I don't think this realistically should be under 0.6, probably more 0.7 or even 0.8 when we focus more on the GUI and UX, thoughts?

**Comment by Zomis** [comment]
Agreed. Postponing until 0.8.
